gulpfile.js

```diff
@@ -1,87 +1,25 @@
-// Include Gulp and Plugins
-var _           = require('underscore');
-var argv        = require('yargs').argv;
-var colors      = require('colors');
-var fm          = require('front-matter');
-var fs          = require('fs');
-var gulp        = require('gulp');
-var gzip        = require('gulp-gzip');
-var handlebars  = require('handlebars');
-var marked      = require('marked');
-var mkdirp      = require('mkdirp');
-var moment      = require('moment');
-var rename      = require('gulp-rename');
-var requireDir  = require('require-dir');
-var rmrf        = require('rimraf');
-var sass        = require('gulp-sass');
+'use strict';
 
-var conf        = require('./conf.json');
-var dir         = requireDir('./tasks/');
+var gulp = require('gulp');
+var tasks = require('./lib/tasks/');
 
+gulp.task('post', function() {
+    var title, slug = "Post title example";
```

Comment:
Not sure why you have set `slug` to the same as the `title`. You either leave it out or set it to something specific . Keep the variables on separate lines for now